src/ni/vsbuild/steps/LvVITesterStep.groovy

```diff
@@ -0,0 +1,18 @@
+package ni.vsbuild.steps
+
+import ni.vsbuild.BuildConfiguration
+
+class LvVITesterStep extends LvProjectStep {
+
+   def Test_path
```

Comment:
`def test_Path` is the variable naming convention for groovy.

Reply:
updated to `def testPath`